Resources/Prototypes/DeltaV/tags.yml

```diff
@@ -65,7 +65,7 @@
 
 - type: Tag
   id: HidesHarpyWings
-  
```

Comment:
the space was *removed* by this pr, realistically not the end of the world but you can always `git checkout usptream/master Resources/Prototypes/DeltaV/tags.yml` for resetting a change you cant see